buildSrc/src/main/java/org/elasticsearch/gradle/PublishPlugin.java

```diff
@@ -116,25 +101,24 @@ private static void addNameAndDescriptiontoPom(Project project, NamedDomainObjec
     }
 
     private static void configureWithShadowPlugin(Project project, MavenPublication publication) {
-        project.getPluginManager().withPlugin("com.github.johnrengelman.shadow", plugin -> {
-            ShadowExtension shadow = project.getExtensions().getByType(ShadowExtension.class);
-            shadow.component(publication);
```

Comment:
Why were we doing this? Is this no longer necessary?

Reply:
we configure shadow jars to have the default naming. with having tis here we would basically add a 2nd artifact which causes gradle to declare the packaging to be a pom as we have already declared the java component as artifact in the withJava block.